rpc/json/rpc.json

```diff
@@ -576,15 +576,19 @@
       "last_txs_updated_at": "0",
       "orphan": "0",
       "pending": "1",
-      "proposed": "0"
+      "proposed": "0",
+      "total_tx_cycles": "2",
+      "total_tx_size": "156"
```

Comment:
Please revert the format changes below, it will cause problems when merged back to develop. I'll set up a CI job to enforce this file is formatted using prettier.